src/utils/format.ts

```diff
@@ -154,7 +151,7 @@ export function mapWhereFieldNames(where: Record<string | symbol, any>, Model: M
  * @returns All keys including operators
  * @private
  */
-export function getComplexKeys(obj: object): Array<string | symbol> {
+export function getComplexKeys(obj: object): PropertyKey[] {
```

Comment:
Note that PropertyKey is `string | symbol | number`, but numbers are stringified when used as a key so this method would only get back `string | symbol`

Reply:
In other cases using PropertyKey is fine right?

Reply:
This is fixed

Reply:
Yep!